spec/model_spec.cr

```diff
@@ -97,4 +120,48 @@ describe LuckyRecord::Model do
       QueryMe.column_names.should eq [:id, :created_at, :updated_at, :email, :age]
     end
   end
+
+  describe ".ensure_correct_field_mappings" do
+    # table is missing
+    it "raises on missing table" do
+      missing_table = MissingTable.new(1, Time.new, Time.new)
+      expect_raises Exception, "The table uusers was not found, did you mean users?" do
```

Comment:
I think switching `,` to a period `.` would be more correct. Also maybe adding the yellow color to `Did you mean 'users'?` would be cool:

```crystal
"The table 'uusers' was not found. #{"Did you mean users?".colorize("yellow")}"
```

Reply:
I tried adding color, but it broke the test by surrounding the colorized text with `\e[33m` and `\e0m`.
```
Expected Exception with "The table 'uusers' was not found. \e[33mDid you mean users?\e[0m", got #<Exception: The table 'uusers' was not found. Did you mean users?
```
Do you know how to handle this?